### PR TITLE
Fix coverage issues

### PR DESCRIPTION
**Pull request by grosspersky**
# Request Title:

## Description:
This PR solves problems with test coverage not being correctly recorded.
The problem was caused by gulp-mocha being incompatible with istanbul.

Calling istanbul natively and adding configurations files for both istanbul and mocha solves the problem.

## Improvements
Coverage is finally working

## Known Issues:
None

### CODE REVIEW

File: .travis/coveralls.sh [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 # Variables
 MODULE_PATH=".travis/node_modules"
-BIN_PATH="coveralls/bin"
+# BIN_PATH="coveralls/bin"
+BIN_PATH=".bin"
```
Comment (aeimer) [inline .travis/coveralls.sh]: Which bin folder do you reference here?

File: .travis/coveralls.sh [hunk 2]
```diff
@@ -29,7 +30,8 @@ fi
 cd ..
 
 echo "+ sending lcov file to coveralls"
-cat api/coverage/lcov.info | $MODULE_PATH/$BIN_PATH/coveralls.js -v
+sed -i 's/bin\///g' api/coverage/lcov.info && cat api/coverage/lcov.info | $MODULE_PATH/$BIN_PATH/coveralls
```
Comment (aeimer) [inline .travis/coveralls.sh]: What does this sed does?

File: .travis/coveralls.sh (hunk 1 above)
Comment (aeimer) [reply .travis/coveralls.sh]: All right, git this one. Could you adapt the same thing here https://github.com/h-da/geli/blob/bugfix/coverage/.travis/license-crawler.sh#L10 ? thx

File: .travis/coveralls.sh (hunk 2 above)
Comment (grosspersky) [reply .travis/coveralls.sh]: Take a look at https://de.wikipedia.org/wiki/Sed_(Unix) ;-)

Comment (aeimer) [reply .travis/coveralls.sh]: Thanks for the detailed explanation...

File: .travis/coveralls.sh (hunk 1 above)
Comment (grosspersky) [reply .travis/coveralls.sh]: Included the fix for the license-crawler.sh

**Pull request merged**